tests/baselines/reference/project/nodeModulesMaxDepthIncreased/amd/nodeModulesMaxDepthIncreased.errors.txt

```diff
@@ -28,11 +29,22 @@ maxDepthIncreased/root.ts(4,1): error TS2322: Type 'string' is not assignable to
     
     exports.f2 = m2;
     
-==== maxDepthIncreased/root.ts (1 errors) ====
+==== entry.d.ts (0 errors) ====
+    export declare var foo: number;
+    
+==== maxDepthIncreased/root.ts (2 errors) ====
     import * as m1 from "m1";
+    import * as m4 from "m4";
+    
     m1.f1("test");
     m1.f2.a = 10;
-    m1.f2.person.age = "10"; // Error: Should be number
+    
+    m1.f2.person.age = "10";    // Should error if loaded the .js files correctly
     ~~~~~~~~~~~~~~~~
 !!! error TS2322: Type 'string' is not assignable to type 'number'.
+    let r1 = m4.test.charAt(2); // Should error if correctly not using the .js file but using @types info
+                ~~~~
+!!! error TS2339: Property 'test' does not exist on type 'typeof "C:/src/TypeScript/tests/cases/projects/NodeModulesSearch/maxDepthIncreased/node_modules/@...'.

```

Comment:
@RyanCavanaugh I'm seeing this be the actual error (with full path) when a types package has an incorrect type. Is this expected? Doesn't seem very readable.

![error](https://cloud.githubusercontent.com/assets/993909/16498587/5bd6b0d6-3eb1-11e6-9978-5ebca736a92f.png)